akka-actor-typed/src/main/scala/akka/actor/typed/internal/InterceptorImpl.scala

```diff
@@ -141,7 +141,7 @@ private[akka] final case class MonitorInterceptor[T](actorRef: ActorRef[T]) exte
  * INTERNAL API
  */
 @InternalApi
-private[akka] final case class LogMessagesInterceptor[T](opts: LogOptions) extends BehaviorInterceptor[T, T] {
+private[akka] final case class LogMessagesInterceptor[T: ClassTag](opts: LogOptions) extends BehaviorInterceptor[T, T] {
```

Comment:
Does this really have to have the class tag, can't it just be `Any` since all it does is logging it?

Reply:
good idea, removed it there, in 0abe9eb867406a6e7fd5f42055cbf675cc37d15d